kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -1,6 +1,124 @@
+        $.ajax({
+            context: this,
+            type: 'post',
+            url: 'api/django_user',
+            dataType: 'json',
+            data : {'intent' : 'addform'},
+            success : function(e){
+                if (e.Status == 'ShowModal'){
+                    $('#superusercreate-container').html(e.data);
+                    setTimeout(function () {this.$("#id_superusername").focus().select()}, 900);
+                }
+            },
+            error : function(e){
+                console.log(e);
+            }
+        });
+    },
+
+    create_superuser_click: function() {
+        $('#superusercreate-box').submit({param1: this}, function (e) {
+            mContext = e.data.param1;
+            e.preventDefault();
+            $.ajax({
+                context: mContext,
+                type: 'post',
+                url: 'api/django_user',
+                dataType: 'json',
+                data: $("#superusercreate-box").serialize(),
+                success : function(e){
+                    if (e.Status == 'Success') {
+                        this.close_modal();
+                    }else if (e.Status == 'Invalid'){
+                        $('#superusercreate-container').html(e.data);
+                        this.highlightForm();
+                    }
+                },
+                error : function(e){
+                    console.log(e);
+                }
+            });
+        });
+    },
+
+    highlightForm: function(){
+        if (this.validateEmail(this.$("#id_superemail").val())){
+            this.$("#id_superemail").css({ 'box-shadow': '0 0 5px 3px rgba(0,171,0,0.75) inset', 'border-color':'#03B3FF'});
+        }else{
+            this.$("#id_superemail").focus().css({ 'box-shadow': '0 0 5px 3px rgba(171,0,0,0.75) inset', 'border-color':'#a94442'});
+        }
+        if (this.$("#id_superpassword").val()){
+            this.$("#id_superpassword").css({ 'box-shadow': '0 0 5px 3px rgba(0,171,0,0.75) inset', 'border-color':'#03B3FF'});
+        }else{
+            this.$("#id_superpassword").focus().css({ 'box-shadow': '0 0 5px 3px rgba(171,0,0,0.75) inset', 'border-color':'#a94442'});
+        }
+        if (this.$("#id_superusername").val()){
+            this.$("#id_superusername").css({ 'box-shadow': '0 0 5px 3px rgba(0,171,0,0.75) inset', 'border-color':'#03B3FF'});
+        }else{
+            this.$("#id_superusername").focus().css({ 'box-shadow': '0 0 5px 3px rgba(171,0,0,0.75) inset', 'border-color':'#a94442'});
+        }
+    },
+
+    validateEmail: function(email){
+        var re = /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
+        return re.test(email);
+    },
+
+    close_modal: function() {
+        $("#superUserCreateModal").modal('hide');
+    },
+
+    show_modal: function() {
+        $("#superUserCreateModal").modal('show');
+    },
+
+    key_user: function(event) {
+        if (event.which == 13) {
```

Comment:
Which keypress does this correspond to? "Enter" right? Can you do something like 

```js
this.ENTER_KEY = 13;
...
if (event.which === this.ENTER_KEY ) {
...
```